### PR TITLE
Update api.md

### PR DESCRIPTION
**Pull request by Bob1k**

### CODE REVIEW

File: api.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ curl -X POST "$GOTRACE_API/v1/orgs/$ORG_ID/events" \
     "entity_id": "eboThjQLWfAd79dvQ05O",
     "entity": "org",
     "deleted_at": null,
-    "type": "form",
```
Comment (treeder) [inline api.md:61]: We should probably have some info on how to submit a form here too.

**Review by treeder — APPROVED** [review]

File: api.md [hunk 2]
```diff
@@ -2194,6 +2190,46 @@ Event type, that is used to add custom fields form to location history (location
 ```
 </details>
 
+## Events
+
+### GET Event by ID (only for organization/location events)
+
+```sh
+curl "$GOTRACE_API/v1/events/$EVENT_ID" \
+  -H "Authorization: Bearer $API_TOKEN" \
```
Comment (treeder) [inline api.md:2199]: Don't end a command with a `\`

File: api.md (hunk 1 above)
Comment (Bob1k) [reply api.md:61]: yes, of course, I just created pr to give them what they've asked

File: api.md (hunk 2 above)
Comment (Bob1k) [reply api.md:2199]: Знімок екрана 2024-02-06 о 20 59 22
half of curl examples in documentation ends with \

Comment (Bob1k) [reply api.md:2199]: I'll remove them

**Pull request merged**